app/models/service_plan.rb

```diff
@@ -10,4 +11,44 @@ class ServicePlan < ApplicationRecord
   has_many   :service_instances
 
   acts_as_taggable
+
+  def order(catalog_id, additional_parameters)
```

Comment:
I feel like all of this should be in a module in the API repo since it's all temporary and nothing else here will ever use it.  It will be much more clear that we should delete all of it in the same PR that does it the newer way.

Reply:
I agree, if we move the rest of the methods to the service_catalog_client module I think that will accomplish this.  We can just add a comment over the `service_catalog_client.order_service_plan(payload)` call stating that this is temporary.